src/UI/WB.UI.Designer/Controllers/QuestionnaireController.Options.cs

```diff
@@ -24,12 +24,12 @@ namespace WB.UI.Designer.Controllers
 {
     public partial class QuestionnaireController
     {
-        public ActionResult<EditOptionsViewModel> GetCategoryOptions(QuestionnaireRevision id, Guid categoriesId)
+        public EditOptionsViewModel? GetCategoryOptions(QuestionnaireRevision id, Guid categoriesId)

```

Comment:
why changes returned status code for request?

Reply:
Incorrect refactoring. Fixed.